src/entries/popup/pages/home/NFTs/NFTDetails.tsx

```diff
@@ -0,0 +1,1039 @@
+              <Text
+                weight="semibold"
+                size="14pt"
+                color="labelTertiary"
+                align="right"
+              >
+                {i18n.t('nfts.details.floor_price')}
+              </Text>
+              <Symbol
+                symbol="info.circle"
+                color="labelTertiary"
+                size={10}
+                weight="semibold"
+              />
+            </Inline>
+            {nft?.floorPriceEth ? (
+              <Text weight="bold" size="14pt" color="label" align="right">
+                {`${nft?.floorPriceEth} ETH`}
+              </Text>
+            ) : (
+              <Text
+                weight="bold"
+                size="14pt"
+                color="labelTertiary"
+                align="right"
+              >
+                {i18n.t('nfts.details.none')}
+              </Text>
+            )}
+          </Stack>
+        </Column>
+      </Columns>
+    </Box>
+  );
+};
+
+const NFTAccordionDescriptionSection = ({
+  nft,
+}: {
+  nft?: UniqueAsset | null;
+}) => {
+  return (
+    <>
+      <AccordionItem value="description">
+        <AccordionTrigger>{'Description'}</AccordionTrigger>
+        <AccordionContent gap="24px">
+          <div />
+          <NFTDescription text={nft?.description} />
+        </AccordionContent>
+      </AccordionItem>
+      <Separator color="separatorTertiary" />
+    </>
+  );
+};
+
+const NFTAccordionTraitsSection = ({
+  traits,
+}: {
+  traits: UniqueAsset['traits'];
+}) => {
+  return (
+    <AccordionItem value="properties">
+      <AccordionTrigger>{i18n.t('nfts.details.properties')}</AccordionTrigger>
+      <AccordionContent gap="24px">
+        <div />
+        <Box
+          style={{
+            display: 'flex',
+            flexDirection: 'row',
+            flexWrap: 'wrap',
+            gap: 6,
+          }}
+        >
+          {traits.map((trait) => (
+            <Box
+              key={trait.trait_type}
+              borderColor="accent"
+              borderWidth="2px"
+              borderRadius="16px"
+            >
+              <Box paddingVertical="12px" paddingHorizontal="10px">
+                <Stack space="6px">
+                  <Text color="labelSecondary" size="11pt" weight="semibold">
+                    {String(trait.trait_type || '').toUpperCase()}
+                  </Text>
+                  <Text color="label" size="14pt" weight="bold">
+                    {String(trait.value || '').toUpperCase()}
+                  </Text>
+                </Stack>
+              </Box>
+            </Box>
+          ))}
+        </Box>
+      </AccordionContent>
+    </AccordionItem>
+  );
+};
+
+const NFTAccordionAboutSection = ({ nft }: { nft?: UniqueAsset | null }) => {
+  const network = useMemo(() => {
+    if (nft?.network === 'mainnet') {
+      return 'Ethereum';
+    }
+    return nft?.network && capitalize(nft.network);
+  }, [nft?.network]);
```

Comment:
I think it's more expensive to `useMemo` this then just
```suggestion
  const network = nft?.network === 'mainnet' ? 'Ethereum' : capitalize(nft.network);
```